test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingHomonymAdditions/GivenMissingHomonymAdditionWithLanguage.cs

```diff
@@ -0,0 +1,71 @@
+namespace StreetNameRegistry.Tests.AggregateTests.WhenCorrectingHomonymAdditions;
+
+using AutoFixture;
+using Be.Vlaanderen.Basisregisters.AggregateSource.Testing;
+using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+using global::AutoFixture;
+using Municipality;
+using Municipality.Commands;
+using Municipality.Events;
+using Municipality.Exceptions;
+using Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+public class GivenMissingHomonymAdditionWithLanguage : StreetNameRegistryTest
```

Comment:
I would rename this class to GivenStreetNameWithoutHomonymAddition. They the starting scenario on which you are going to correct a homonym addition is more clear.